integration_tests/test_domestic_migration.py

```diff
@@ -0,0 +1,39 @@
+import pytest
+
+# TODO: Broader test coverage
+
+
+def test_individuals_do_move(populations):
+    for before, after in zip(populations, populations[1:]):
+        common_simulants = before.index.intersection(after.index)
+        before = before.loc[common_simulants]
+        after = after.loc[common_simulants]
+
+        individual_movers = before["household_id"] != after["household_id"]
+        assert individual_movers.any()
+        assert (
+            before[individual_movers]["address_id"] != after[individual_movers]["address_id"]
+        ).all()
+
+
+def test_households_do_move(populations):
+    for before, after in zip(populations, populations[1:]):
+        common_simulants = before.index.intersection(after.index)
+        before = before.loc[common_simulants]
+        after = after.loc[common_simulants]
+
+        household_movers = (before["household_id"] == after["household_id"]) & (
+            before["address_id"] != after["address_id"]
+        )
+        assert household_movers.any()
+
+
+def test_only_living_people_move(populations):
+    for before, after in zip(populations, populations[1:]):
+        common_simulants = before.index.intersection(after.index)
+        before = before.loc[common_simulants]
+        after = after.loc[common_simulants]
+
+        movers = before["address_id"] != after["address_id"]
+        assert after[movers]["tracked"].all()
```

Comment:
What if there were no dead people at all? You should probably assert that there are some people that are dead who could have moved.

Reply:
Hmm... I was thinking that the test here was that the component only moves living simulants; that people die in the sim is a pretty basic assumption that IMO would go in a test of the mortality component. But, if that doesn't sound right to you I can add the assert here.

Reply:
No I think you're right, we should test that independently